mkdocs/docs/getting-started/quick-start.md

```diff
@@ -102,6 +104,12 @@ Choose the K8s stack over which you are performing the A/B testing experiment.
         ```shell
         $ITER8/samples/knative/quickstart/platformsetup.sh istio
         ```
+=== "Seldon"
+    Setup Seldon Core, Seldon Analytics and Iter8 within your cluster.
+
+    ```shell
+    $ITER8/samples/seldon/quickstart/platformsetup.sh
```

Comment:
I am looking into the `platformsetup.sh` more closely...

```shell
## 0(d). Ensure Helm is available
KUSTOMIZE_VERSION=$(helm  version | cut -d. -f2 | tail -c 2)
if [[ ${KUSTOMIZE_VERSION} -ge "3" ]]; then
    echo "Helm v3+ available"
else
    echo "Helm v3+ is unavailable"
    exit 1
fi
```

Could you please replace `KUSTOMIZE` above with `HELM`.

Reply:
yes